intelmq/bots/BOTS

```diff
@@ -500,6 +500,13 @@
             "module": "intelmq.bots.experts.gethostbyname.expert",
             "parameters": {}
         },
+        "IDEA": {
+            "description": "Converts events into the IDEA format.",
+            "module": "intelmq.bots.experts.idea.expert",
+            "parameters": {
+                "test_mode": false
```

Comment:
It does not change, it _appends_ additional category (in Idea there is possibility for event to have multiple categories). It is handy when setting up new client - all outgoing events are as they should, however they are also marked as "Test", so if they end up in database or recipients, everybody is able to easily filter them out.